site/src/components/components/PropsTable.module.css

```diff
@@ -0,0 +1,3 @@
+.tableWrapper td > * {
+  margin-bottom: 0;
```

Comment:
Is this to compensate `p` in a cell? I wonder whether `p` should be used ..?

Or if there is genuine need for multi-paragraph support, should margin be removed?

Reply:
That's a good point, the `p` is actually added by `ReactMarkdown` and I can replace it with something else like a `span` but then it wouldn't support multi paragraph very well so I'll just restore the margin